content/realtime/channels/channel-parameters/overview.textile

```diff
@@ -0,0 +1,82 @@
+---
+title: Channel Parameters
+section: realtime
+index: 1
+jump_to:
+  Help with:
+    - Overview#overview
+    - Supported channel parameters#supported-parameters
+    - Using channel parameters#using-parameters
+    - Using channel parameters with non-Ably transports#using-parameters-with-other-transports
+    - Next steps#next-steps
+---
+
+h2(#overview). Overview
+
+Channel parameters allow a client to set the properties of a channel. A set of channel parameters is a set of key/value pairs, where both keys and values are strings; the keys correspond to specific features that Ably defines.
+
+The methods provided for specifying channel parameters are outlined below.
+
+h2(#supported-parameters). Supported channel parameters
+
+- rewind := Used to request that an attachment start from a given number of messages or point in time in the past. See "rewind":/realtime/channels/channel-parameters/rewind for more information.
+- delta := Used to request that data payloads should be sent as deltas to the previous payload. See "deltas":/realtime/channels/channel-parameters/deltas for more information.
```

Comment:
I appreciate this is a list of parameters, but I think the overview of parameters should be more explicit about what parameters are used for i.e. we currently support two key features, rewind which is X, and Delta Compression, which is Y.  Also, we refer to this as deltas here, yet refer to it elsewhere as Delta Compression.

Reply:
I think the overview should remain generic, otherwise, it could become a long-winded section of parameters that just repeats itself once you get to this point. But we can address the need for channel parameters as discussed on a previous comment.

I think the term deltas here is correct, delta compression is the overall feature name but you send as 'deltas' to previous payloads. Unless you disagree @paddybyers?

Reply:
True. I commented on this out of context.